src/Aquarius.ONE.ClientSDK/Aquarius.ONE.ClientSDK.csproj

```diff
@@ -23,7 +23,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Aquarius.ONE.Models.CSharp" Version="6.11.0" />
+    <PackageReference Include="Aquarius.ONE.Models.CSharp" Version="6.13.0-AQI-7691" />

```

Comment:
Make sure this isn't merged with the pre-release version

Reply:
This has been resolved @AQI-DanG  @MatthewDeRose-AI